cypress/e2e/alerts.cy.js

```diff
@@ -1,25 +1,47 @@
 describe('Cypress application', () => {
-  before(() => {
-
+  beforeEach(() => {
+    cy.visit('https://demoqa.com/alerts');
   });
 
   it('should have the ability to assert automatically resolved alerts', () => {
+    cy.get('#alertButton').click();
 
+    cy.on('window:alert', (alertText) => {
+      expect(alertText).to.equal('You clicked a button');
+    });
   });
 
   it('should have the ability to assert scheduled allert', () => {
+    cy.get('#timerAlertButton').click();
+
+    cy.wait(5000);
 
+    cy.on('window:alert', (alertText) => {
+      expect(alertText).to.equal('This alert appeared after 5 seconds');
+    });
   });
 
   it('should autimatically resolve alerts', () => {
+    cy.get('#confirmButton').click();
+
+    cy.on('window:confirm', (confirmText) => {
+      expect(confirmText).to.equal('Do you confirm action?');
 
+      return true;
```

Comment:
by default its returns true and confirm alert
 unnecessary

Reply:
it is not critical